### PR TITLE
build: provide development module for easy PostgreSQL setup on NixOS

### PR DESCRIPTION
**Pull request by erictapen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +31,18 @@ Or set up [`nix-direnv`](https://github.com/nix-community/nix-direnv) on your sy
...
 
-```
-services.postgresql.enable = true;
-services.postgresql.ensureDatabases = [ "nix-security-tracker" ];
-services.postgresql.ensureUsers = [{
-  name = "nix-security-tracker";
-  ensureDBOwnership = true;
-}];
+```nix
+{
```
Comment (fricklerhandwerk) [inline README.md:35]:
```suggestion
{ ... }:
{
```

Make clear it's a module

**Review by fricklerhandwerk — APPROVED** [review]

File: nix/dev-setup.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    services.postgresql = {
+      enable = true;
+      ensureDatabases = [ "nix-security-tracker" ];
+      ensureUsers = [
+        {
+          name = "nix-security-tracker";
+          ensureDBOwnership = true;
+        }
+      ];
+      extraPlugins = with config.services.postgresql.package.pkgs; [ postgis ];
```
Comment (RaitoBezarius) [inline nix/dev-setup.nix:34]: Hm, do we need PostGIS for something?

**Comment by RaitoBezarius** [comment]
Ah it was just merged.

File: nix/dev-setup.nix (hunk 2 above)
Comment (erictapen) [reply nix/dev-setup.nix:34]: Oh my bad, that was copied from something else!

**Pull request merged**